### ISSUE TITLE
Use aria-hidden="true" and tabindex=“-1" to avoid repeating duplicate links for screen readers

### ISSUE THREAD

**Issue by alice-blue** [iftechfoundation/ifdb#1377]
On various pages, IFDB has lists of games where each game has a cover art thumbnail that links to the page for that game, and nearby, the game's title linking to the same place. This kind of thing can cause redundancy for screen readers:

> Otherwise what happens is when you’re browsing you hear “Cover of foo, graphic link” and then “foo” when you move to the next item which is the plain text link. Ideally you only want to hear one of those, the other is extraneous.

 ([Discussed here](https://intfiction.org/t/poll-alt-text-for-cover-art-on-ifdb/80337/7))

Right now, it looks like the alt text is set to "" for the thumbnails, but apparently this can still be an inconvenience when the image is the only thing in a link. According to [Webaim.org](https://webaim.org/techniques/alttext/#functional):

> When an image is the only content inside a link or button, alt text is all that a screen reader has to go on. If it’s empty or missing, screen readers might read the image filename or the URL of the linked page in hopes that this may be helpful to the user, but this is not always the case.

An IFDB user suggested dealing with duplicate links by either combining the two links into one, or using aria-hidden=“true” and tabindex=“-1” on the `<a>` element on the image link. Here is a [similar recommendation from accessibleweb.com](https://accessibleweb.com/question-answer/what-should-i-do-if-i-have-multiple-links-that-go-to-the-same-page/ ).

Since combining the links isn't always practical, for example, when the links are in different cells of a table, I suggest using aria-hidden="true" and tabindex=“-1" in cases where combining the links doesn't make sense.

(For the suggestion about combining adjacent links, see https://github.com/iftechfoundation/ifdb/issues/1374 )
